javascript-modal/main.js

```diff
@@ -0,0 +1,13 @@
+var $openButton = document.querySelector('.open-button');
+var $modal = document.querySelector('.modal-off');
+var $noButton = document.querySelector('.no-button');
+
+function handleClick(event) {
+  $modal.className = 'modal-on';
```

Comment:
I would make this line an `else` statement for the conditional you have below this, so that you only show the modal if you are meant to show the modal.